apps/omg/test/support/test_helper.ex

```diff
@@ -154,10 +154,10 @@ defmodule OMG.TestHelper do
   @doc """
   Always creates file in the priv/ folder of the application.
   """
-  @spec write_fee_file(%{Crypto.address_t() => map()} | binary(), binary() | nil) :: {:ok, binary, binary}
-  def write_fee_file(fee_map, file_name \\ nil)
+  @spec write_fee_file(%{Crypto.address_t() => map()} | binary()) :: {:ok, binary}
+  def write_fee_file(fee_map, file_path \\ nil)
 
-  def write_fee_file(map, file_name) when is_map(map) do
+  def write_fee_file(map, file_path) when is_map(map) do
```

Comment:
why does this first parameter keep on changing it's name?
`fee_map`, `map`, `content`?

Reply:
🤦 renamed `fee_map` to a more accurate `map_or_content`